apps/frontend/src/pages/PurposeListPage.tsx

```diff
@@ -28,6 +28,12 @@ export const PurposeListPage = () => {
       setShowCreateProcessModal(false)
       // todo multipurpose url
       history.push(genProcessPath(Section.purpose, newProcess.purposes[0].code, newProcess, undefined, true))
+      process.disclosures.forEach(d=>{
+        updateDisclosure(convertDisclosureToFormValues(
+          {...d,processIds:[...d.processIds,newProcess.id?newProcess.id:'']}
```

Comment:
Ikke helt heldig å legge til en tom streng her, backend filtrerer den ut for deg I take it?